google/cloud/forseti/common/gcp_type/iam_policy.py

```diff
@@ -348,6 +348,10 @@ def matches(self, other):
                 other_member.type == self.ALL_USERS):
             return True
 
+        if (self.type == self.ALL_AUTH_USERS and
```

Comment:
Can you please update the comment in line 345 to include `allAuthenticatedUsers`.

```
Bucket IAM supports special "allUsers" and "allAuthenticatedUsers" members,
whose values are simply"allUsers" and "allAuthenticatedUsers",
without a colon separator and a second fragment.

Reply:
Done